### PR TITLE
added 3 more java questions

### PR DESCRIPTION
**Pull request by KeysGilbert**
## What type of PR is this? (check all applicable)

- [ ] 🚀 Added Name
- [ ] ✨ Feature
- [ ] 🌟 stared the repo
- [ ] 🐛 Grammatical Error
- [X ] 📝 Documentation Update
- [ ] 🚩 Other

## Description
Edited questions.json in Java folder.

### CODE REVIEW

File: Languages/Java/questions.json [hunk 1]
```diff
@@ -71,6 +71,34 @@
...
+      "shortAns": "No",
+      "tag": "Java"
+    },
+    {
+      "id": 10,
+      "question":"What does the 'synchronized' keyword do?",
+      "details": "'synchronized' isolates a thread so that other threads using the same resource(s) cannot interrupt it.",
+      "questionType": "multi",
+      "trueOrFalse": false,
+      "shortAns": "It isolates a thread.",
```
Comment (Tc2r1) [inline Languages/Java/questions.json:99]: Please phrase short answers as "It is" etc.

**Review by Tc2r1 — CHANGES_REQUESTED** [review]
Please  make requested changes.

**Review by AndroidLion48 — APPROVED** [review]
Well Done! Please do not forget to add your name to CONTRIBUTERS.MD list and star the repository. @KeysGilbert

**Review by Tc2r1 — APPROVED** [review]
awaiting you to add name to contributors list.

**Pull request merged**